src/libs/HttpUtils.js

```diff
@@ -22,6 +22,13 @@ Onyx.connect({
 // We use the AbortController API to terminate pending request in `cancelPendingRequests`
 let cancellationController = new AbortController();
 
+/**
+ * API comamnds we need to calculate skew,
```

Comment:
nitpick: I would separate and add a comment for each of the variables. Also, small typo and construction of the phrase

```suggestion
 * The API commands that require the skew calculation
```